libensemble/tests/scaling_tests/forces/forces_adv/readme.md

```diff
@@ -71,8 +71,7 @@ Then to test locally, run the `balsam_local.sh` script. The default runs with 2
 
 The running tasks can be seen inside the balsam database dir `<DIR>/data/libe_workflow/`.
 While the key output files will be copied back to the run dir at completion. Also see
-the log in `<DIR>/log` if there are any issues. To run on batch systems, see the example
-scripts such as `theta_submit_balsam.sh`.
```

Comment:
Does the cobalt script work here

Reply:
Yeah, it could go in the Using batch scripts section.

But looking at the Balsam content, I think I should remove it. The LegacyBalsamExecutor is gone, `USE_BALSAM=True` is gone, `balsam_local.sh` is gone, and databases are gone of course.